_deno_unstable.ts

```diff
@@ -0,0 +1,143 @@
+  ...args: Parameters<typeof Deno.createHttpClient>
+): ReturnType<typeof Deno.createHttpClient> {
+  if (typeof Deno.createHttpClient == "function") {
+    return Deno.createHttpClient(...args);
+  } else {
+    throw new TypeError("Requires --unstable");
+  }
+}
+
+export function consoleSize(
+  ...args: Parameters<typeof Deno.consoleSize>
+): ReturnType<typeof Deno.consoleSize> {
+  if (typeof Deno.consoleSize == "function") {
+    return Deno.consoleSize(...args);
+  } else {
+    throw new TypeError("Requires --unstable");
+  }
+}
+
+export function futime(
+  ...args: Parameters<typeof Deno.futime>
+): ReturnType<typeof Deno.futime> {
+  if (typeof Deno.futime == "function") {
+    return Deno.futime(...args);
+  } else {
+    return Promise.reject(new TypeError("Requires --unstable"));
+  }
+}
+
+export function futimeSync(
+  ...args: Parameters<typeof Deno.futimeSync>
+): ReturnType<typeof Deno.futimeSync> {
+  if (typeof Deno.futimeSync == "function") {
+    return Deno.futimeSync(...args);
+  } else {
+    throw new TypeError("Requires --unstable");
+  }
+}
+
+export function hostname(
+  ...args: Parameters<typeof Deno.hostname>
+): ReturnType<typeof Deno.hostname> {
+  if (typeof Deno.hostname == "function") {
+    return Deno.hostname(...args);
+  } else {
+    throw new TypeError("Requires --unstable");
+  }
+}
+
+export function loadavg(
+  ...args: Parameters<typeof Deno.loadavg>
+): ReturnType<typeof Deno.loadavg> {
+  if (typeof Deno.loadavg == "function") {
+    return Deno.loadavg(...args);
+  } else {
+    throw new TypeError("Requires --unstable");
+  }
+}
+
+export function osRelease(
+  ...args: Parameters<typeof Deno.osRelease>
+): ReturnType<typeof Deno.osRelease> {
+  if (typeof Deno.osRelease == "function") {
+    return Deno.osRelease(...args);
+  } else {
+    throw new TypeError("Requires --unstable");
+  }
+}
+
+export function removeSignalListener(
+  ...args: Parameters<typeof Deno.removeSignalListener>
+): ReturnType<typeof Deno.removeSignalListener> {
+  if (typeof Deno.removeSignalListener == "function") {
+    return Deno.removeSignalListener(...args);
+  } else {
+    throw new TypeError("Requires --unstable");
+  }
+}
+
+export function setRaw(
+  ...args: Parameters<typeof Deno.setRaw>
+): ReturnType<typeof Deno.setRaw> {
+  if (typeof Deno.setRaw == "function") {
+    return Deno.setRaw(...args);
+  } else {
+    throw new TypeError("Requires --unstable");
+  }
+}
+
+export function systemMemoryInfo(
+  ...args: Parameters<typeof Deno.systemMemoryInfo>
+): ReturnType<typeof Deno.systemMemoryInfo> {
+  if (typeof Deno.systemMemoryInfo == "function") {
+    return Deno.systemMemoryInfo(...args);
+  } else {
+    throw new TypeError("Requires --unstable");
+  }
+}
+
+export function umask(
+  ...args: Parameters<typeof Deno.umask>
+): ReturnType<typeof Deno.umask> {
+  if (typeof Deno.umask == "function") {
+    return Deno.umask(...args);
+  } else {
+    throw new TypeError("Requires --unstable");
```

Comment:
`Deno.umask requires --unstable` would be a better error message IMO

Reply:
I'm not sure, the intended subject in this case is whatever API at the lib surface e.g. `copy()`. I figure as normal we should let the error stack relay this since it's not a static error.